services/services.go

```diff
@@ -1510,16 +1510,23 @@ func getBurnPageData() (*types.BurnPageData, error) {
 	latestEpoch := LatestEpoch()
 	latestBlock := LatestEth1BlockNumber()
 
+	// Check db to have at least one entry (will error otherwise anyway)
+	burnedFeesCount := 0
+	if err := db.ReaderDb.Get(&burnedFeesCount, "SELECT COUNT(*) FROM chart_series WHERE indicator = 'BURNED_FEES';"); err != nil {
+		return nil, fmt.Errorf("error get BURNED_FEES count from chart_series: %w", err)
+	}
+	if burnedFeesCount <= 0 {
+		return data, nil
+	}
+
 	// Retrieve the total amount of burned Ether
-	err := db.ReaderDb.Get(&data.TotalBurned, "select sum(value) from chart_series where indicator = 'BURNED_FEES';")
-	if err != nil {
-		return nil, fmt.Errorf("error retrieving total burned amount from chart_series table: %v", err)
+	if err := db.ReaderDb.Get(&data.TotalBurned, "SELECT SUM(value) FROM chart_series WHERE indicator = 'BURNED_FEES';"); err != nil {
+		return nil, fmt.Errorf("error retrieving total burned amount from chart_series table: %w", err)
 	}
 
 	cutOff := time.Time{}
-	err = db.ReaderDb.Get(&cutOff, "select (select max(time) from chart_series where indicator = 'BURNED_FEES') + interval '24 hours';")
-	if err != nil {
-		return nil, fmt.Errorf("error retrieving cutoff date from chart_series table: %v", err)
+	if err := db.ReaderDb.Get(&cutOff, "SELECT ( SELECT MAX(time) FROM chart_series WHERE indicator = 'BURNED_FEES' ) + interval '24 hours';"); err != nil {
```

Comment:
Remove the `;` at the end of the sql statement, as we never use it, to be consistent